DiscordBot/Data/InitializeGuilds.cs

```diff
@@ -97,7 +97,8 @@ private User Map(SocketGuildUser socketGuildUser)
             {
                 UserDiscordId = socketGuildUser.Id,
                 DiscriminatorValue = socketGuildUser.DiscriminatorValue,
-                Username = socketGuildUser.Username
+                Username = socketGuildUser.Username,
+                IsBot = socketGuildUser.IsBot
```

Comment:
Something we may want to think about for future features is tracking what bots are being used on a server and how frequently users are interacting with them. It could give us some ideas of functionality to bake into our bot. And also we could serve up that data on the dashboard so that an admin could see what kind of engagement they are getting for each of their bots.

Reply:
That's a really good idea!